src/ResponseFactory.php

```diff
@@ -9,7 +9,6 @@ class ResponseFactory implements ResponseFactoryInterface
 {
     public function createResponse($code = 200)
     {
-        return (new Response())
-            ->withStatus($code);
+        return new Response('php://memory', $code);
```

Comment:
If their default changes to something undesirable in the future we will be happy we wrote `'php://memory'` here. 

> Having one less object isn't worth it in my opinion.

That is fair. Let's close this PR then.